.github/workflows/pr.yml

```
@@ -0,0 +1,72 @@
+name: CI pipeline
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test-codepropertygraph:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-18.04
```
just a suggestion though - personally i'm in favor of controlling the environment myself, i.e. avoid surprising version updates by other parties. same goes for other versions in this file of course

---

Yes, good catch!